### PR TITLE
update client

### PR DESCRIPTION
**Pull request by sklmx**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,68 +1,39 @@
...
-        {
-            "name": "Kamil Samigullin",
-            "email": "kamil@samigullin.info"
-        },
-        {
-            "name": "Sergey Skaridov",
-            "email": "s.skaridov@gmail.com"
-        }
-    ],
+    "version": "1.0.0",
```
Comment (kamilsk) [inline composer.json]: disagree with these changes

**Review by kamilsk — CHANGES_REQUESTED** [review]
how did you generate it?

File: composer.json (hunk 1 above)
Comment (sklmx) [reply composer.json]: fixed, left only updated deps

File: docs/Api/SuitesApi.md [hunk 2]
```diff
@@ -276,7 +276,7 @@ Name | Type | Description  | Notes
 ## `updateSuite()`
 
 ```php
-updateSuite($code, $id, $suiteCreate): \Qase\Client\Model\IdResponse
+updateSuite($code, $id, $inlineObject1): \Qase\Client\Model\IdResponse
```
Comment (kamilsk) [inline docs/Api/SuitesApi.md:279]: looks weird

File: docs/Model/InlineObject1.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# # InlineObject1
```
Comment (kamilsk) [inline docs/Model/InlineObject1.md:1]: is it possible to make it meaningful?

File: docs/Model/Response.md [hunk 4]
```diff
@@ -5,6 +5,5 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **status** | **bool** |  | [optional]
-**result** | **object** |  | [optional]
```
Comment (kamilsk) [inline docs/Model/Response.md:8]: where is result now?

File: docs/Model/Result.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Name | Type | Description | Notes
 **status** | **string** |  | [optional]
 **isApiResult** | **bool** |  | [optional]
 **timeSpentMs** | **int** |  | [optional]
-**endTime** | **\DateTime** |  | [optional]
+**endTime** | **string** | date in the &#39;Y-m-d H:i:s&#39; format | [optional]
```
Comment (kamilsk) [inline docs/Model/Result.md:16]: we discussed about it at March 25th

our decision was to make a research first, and then fix the api.

File: docs/Model/Run.md [hunk 6]
```diff
@@ -9,8 +9,8 @@ Name | Type | Description | Notes
 **description** | **string** |  | [optional]
 **status** | **int** |  | [optional]
 **statusText** | **string** |  | [optional]
-**startTime** | **\DateTime** |  | [optional]
-**endTime** | **\DateTime** |  | [optional]
+**startTime** | **string** | date in the &#39;Y-m-d H:i:s&#39; format | [optional]
+**endTime** | **string** | date in the &#39;Y-m-d H:i:s&#39; format | [optional]
```
Comment (kamilsk) [inline docs/Model/Run.md:13]: the same

File: phpunit.xml.dist [hunk 7]
```diff
@@ -1,25 +1,18 @@
...
-    <testsuites>
-        <testsuite name="tests">
-            <directory>./test/Api</directory>
-            <directory>./test/Model</directory>
-        </testsuite>
-    </testsuites>
-    <php>
-        <ini name="error_reporting" value="E_ALL"/>
-    </php>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" stopOnFailure="false" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```
Comment (kamilsk) [inline phpunit.xml.dist:2]: we lost formatting, let's restore

File: src/ApiException.php [hunk 8]
```diff
@@ -65,10 +65,10 @@ class ApiException extends Exception
     /**
      * Constructor
      *
-     * @param string $message Error message
-     * @param int $code HTTP status code
-     * @param string[]|null $responseHeaders HTTP response header
-     * @param \stdClass|string|null $responseBody HTTP decoded body of the server response either as \stdClass or string
+     * @param string                $message         Error message
```
Comment (kamilsk) [inline src/ApiException.php:68]: did you apply psr formatting after code generation?

File: src/Model/Filters.php [hunk 9]
```diff
@@ -18,7 +18,7 @@
  * The version of the OpenAPI document: 1.0.0
```
Comment (kamilsk) [inline src/Model/Filters.php:18]: could you find possibility to make Filters, Filters1, ... with meaningful name, please?

File: src/Model/InlineObject1.php [hunk 10]
```diff
@@ -0,0 +1,413 @@
...
+ *
+ * @category Class
+ * @package  Qase\Client
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ * @implements \ArrayAccess<TKey, TValue>
+ * @template TKey int|null
+ * @template TValue mixed|null
+ */
+class InlineObject1 implements ModelInterface, ArrayAccess, \JsonSerializable
```
Comment (kamilsk) [inline src/Model/InlineObject1.php:46]: the same here

File: test/Api/RunsApiTest.php [hunk 11]
```diff
@@ -50,11 +49,6 @@ class RunsApiTest extends TestCase
      */
     public static function setUpBeforeClass(): void
     {
-        foreach (['QASE_PROJECT_CODE', 'QASE_API_BASE_URL', 'QASE_API_TOKEN',] as $parameter) {
```
Comment (kamilsk) [inline test/Api/RunsApiTest.php:53]: did you see you lost something?

**Review by kamilsk — CHANGES_REQUESTED** [review]

**Comment by kamilsk** [comment]
it will be great if you extend this https://github.com/qase-tms/specs/blob/a30d45244bbb30e7085c9ea09f8d504987a0af8c/Taskfile#L52

**Comment by kamilsk** [comment]
also, if we going to make it stable, please try to make it better as possible

**Comment by kamilsk** [comment]
also, I sent you security issue

File: docs/Model/Response.md (hunk 4 above)
Comment (sklmx) [reply docs/Model/Response.md:8]: removed recently https://github.com/qase-tms/specs/pull/84

File: docs/Model/Result.md (hunk 5 above)
Comment (sklmx) [reply docs/Model/Result.md:16]: fixed

File: docs/Model/Run.md (hunk 6 above)
Comment (sklmx) [reply docs/Model/Run.md:13]: fixed

File: docs/Api/SuitesApi.md (hunk 2 above)
Comment (sklmx) [reply docs/Api/SuitesApi.md:279]: fixed

File: phpunit.xml.dist (hunk 7 above)
Comment (sklmx) [reply phpunit.xml.dist:2]: fixed

File: docs/Model/InlineObject1.md (hunk 3 above)
Comment (sklmx) [reply docs/Model/InlineObject1.md:1]: fixed

File: src/Model/Filters.php (hunk 9 above)
Comment (sklmx) [reply src/Model/Filters.php:18]: I'll take this later, coulldn't find a way to set a meaningful name

**Pull request closed without merging**